spec/support/ddf/valid_service_plan_ddf.json

```diff
@@ -0,0 +1,103 @@
+{
+  "schema": {
```

Comment:
Does this example exhibit every type of validator option handled in the ddf validator class? I noticed `textarea-field` is missing. I would probably be best to validate against every type.

Reply:
Upon adding more it exposed more bugs with the service plan validation - going to continue that work in another PR since it doesn't pertain here.